topological_navigation/test/test_navigationcore.py

```diff
@@ -94,8 +94,8 @@ def __init__(self, name='test_node'):
 
     def initialize(self):
         self.topmap_sub = self.create_subscription(String, '/topological_map_2', self.topmap_sub_callback, 1)
-        self.closest_node_sub = self.create_subscription(String, 'closest_node', self.closest_node_callback, 1)
-        self.current_node_sub = self.create_subscription(String, 'current_node', self.current_node_callback, 1)
+        self.closest_node_sub = self.create_subscription(String, '/closest_node', self.closest_node_callback, 1)
```

Comment:
see below, no absolute topic URI please change

Reply:
Changed it back to relative topic URIs.